src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Common/AdhocLanguageServices.cs

```diff
@@ -4,9 +4,8 @@
 using System;
 using System.Collections.Immutable;
 using Microsoft.CodeAnalysis.Host;
-using Microsoft.CodeAnalysis.Razor;
 
-namespace Microsoft.AspNetCore.Razor.LanguageServer.Common;
+namespace Microsoft.CodeAnalysis.Razor.Common;
```

Comment:
The "Common" folder and namespace was just an implementation detail after I folded "Microsoft.AspNetCore.Razor.LanguageServer.Common" into "Microsoft.AspNetCore.Razor.LanguageServer". It's probably better to just move these next to `IAdhowWorkspaceFactory` and give them the same namespace and get rid of the "Common" folder/namespace altogether.